content/en/docs/setup/upgrade/helm/canaryupgrade_test.sh

```diff
@@ -13,13 +14,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 source "content/en/docs/setup/upgrade/helm/common.sh"
+source "content/en/boilerplates/snips/args.sh"
 
 set -e
 set -u
 
 set -o pipefail
 
-# @setup profile=none
+# @setup profile=non
```

Comment:
```suggestion
# @setup profile=none
```

Reply:
addressed in 52f8300bf1c5262d40d69f12b36317cd5b4c9fee